components/Article/Article.module.scss

```diff
@@ -14,13 +14,22 @@
   display: -webkit-box;
   -webkit-line-clamp: 3;
   -webkit-box-orient: vertical;
-  overflow: hidden;
+
```

Comment:
Why do we need remove this line

Reply:
Our code tag is bigger than it  parent element .content class  So there was  cut in overflow:hidden

Reply:
<img width="818" alt="image" src="https://user-images.githubusercontent.com/55176773/221575246-8df87504-b2d6-4e51-aa3f-50b74a71532a.png">

here is why we need overflow: hidden attribute
find another way for fixing this